Dockerfile

```diff
@@ -19,6 +19,12 @@ RUN Rscript -e 'install.packages("purr")'
 RUN Rscript -e 'install.packages("flashClust")'
 RUN Rscript -e 'install.packages("data.table")'
 RUN Rscript -e 'install.packages("BiocManager")'
+RUN Rscript -e 'install.packages("RANN")'

```

Comment:
I'd be in favor of dropping this file, I do not expect us to keep maintaining it since we do not intend to use it